scripts/globals/mobskills.lua

```diff
@@ -271,6 +271,10 @@ xi.mobskills.mobPhysicalMove = function(mob, target, skill, numberofhits, accmod
         skill:setMsg(xi.msg.basic.SKILL_MISS)
     end
 
+    if target:getMod(xi.mod.PET_DMG_TAKEN_PHYSICAL) ~= 0 then
+        finaldmg = finaldmg + (finaldmg * (target:getMod(xi.mod.PET_DMG_TAKEN_PHYSICAL) / 100))
```

Comment:
Also these calcs imo should be:

finaldmg = finaldmg * mod

since we want the mod:
to be in scale with default at 100%

so 80% would mean the calc:

dmg = dmg*(mod/100) would me we have dmg*0.8 -> giving us 80% of original dmg

same for 120%

dmg = dmg*(mod/100) would give us dmg*1.2 would give us 120% of original dmg

Reply:
setMod(xi.mod.PET_DMG_TAKEN_PHSY, 120)

Reply:
I mean this way works, but sure? I'll go ahead and change it.